guides/taxation/displaying-prices.md

```diff
@@ -0,0 +1,89 @@
+# Displaying prices
+
+Solidus allows you to display prices in a flexible way to comply with your local
+tax regulations.
+
+For example, in most countries that use value-added taxes (VAT), prices in a
+store's frontend need to include any taxes that will be charged to the customer.
+
+## Set up price display for value-added taxes (VAT)
+
+In order to comply with VAT-style tax regulations, you first need to set up the
+tax rates, tax categories, and zones that your store will use in production.
+
+After you have set up and tested your store's tax configuration options, you can
+update any of your existing products and variants with the "Rebuild VAT prices"
+checkbox selected.
+
+Solidus can then proceed to calculate the correct consumer prices for each
+country with `included_in_price` VAT rates. It also generates a fallback
+"export" price, where the price's `country_iso` is `nil`. New products will
+behave as if that checkbox were selected.
```

Comment:
`that checkbox` I believe refers to `Rebuild VAT prices`, correct? Maybe spell it out, we're two sentences away from the last checkbox, and I think `included_in_price` can also be displayed as a checkbox.